src/widgets/refinement-list/refinement-list.js

```diff
@@ -134,10 +134,11 @@ function refinementList({
       return createURL(state.toggleRefinement(attributeName, facetValue));
     }
 
-    // Pass count of currently selected items to the header template
+    // Pass count of total and currently selected items to the header template
+    const facetsCount = facetValues.length;
```

Comment:
`facetValues.length` is the number of values that are retrieved from Algolia. (if an attribute has 3  values then it will be 3). Won't the sum of counts be more interesting?